.github/dependabot.yml

```diff
@@ -13,9 +16,10 @@ updates:
       - "dependencies"
     open-pull-requests-limit: 5
     ignore:
-      # we will always manually update ES dependencies as we need to sync with the other teams
-      # before doing that
+      # we will always manually update ES dependencies to minor or major releases
+      # as we need to sync with the other teams before doing that
       - dependency-name: org.elasticsearch.client:*
+        update-types: [ "version-update:semver-major", "version-update:semver-minor" ]
```

Comment:
I just noticed this and would argue patch version updates are still fine to automate, wdyt?

Reply:
👍 I agree with @npepinpe - please verify this with the other teams.

Reply:
Thanks for raising that, I will reach out to them, I just checked the alignment:

- As of now it seems Operate already moved [forward using 7.17.8 in December](https://github.com/camunda/operate/commit/5b8e4c4e0352b5e60f9897fad897a1e4a1cd3848) updated by dependabot, [the operate dependabot config](https://github.com/camunda/operate/blob/master/.github/dependabot.yml) is not excluding elastic updates
- Optimize [sits on 7.17.3](https://github.com/camunda/camunda-optimize/blob/master/pom.xml#L43), the optimize dependabot setup is[ excluding elastic updates like the zeebe setup](https://github.com/camunda/camunda-optimize/blob/master/.github/dependabot.yml#L17)

Reply:
Both Optimize and Operate are fine with the following agreement:
- automated elastic patch updates for main & stable branches
- agreement for minor/major updates on product wide level